src/js/redux/action-types.js

```diff
@@ -1,3 +1,5 @@
+export const TOGGLE_ACTION = '@@chrome/toggle';
```

Comment:
&lt;nitpick&gt;This is quite ambiguous as it is not clear which toggle this is. Might be helpful to be more specific